src/com/android/launcher3/DeviceProfile.java

```diff
@@ -795,7 +797,7 @@ private void updateHotseatSizes(int hotseatIconSizePx) {
         // radius.
         hotseatCellHeightPx = getIconSizeWithOverlap(hotseatIconSizePx);
 
-        var space = Math.abs(hotseatCellHeightPx / 2);
+        var space = Math.abs(hotseatCellHeightPx / 2) - 10;
```

Comment:
Should this only apply if the dock is disabled?

Reply:
if enable, just a little icons alignment